apps/sms/desktop-mock/navigator_moz_mobilemessage_shim.js

```diff
@@ -7,12 +7,12 @@
 
 *********************************************************** */
 (function(window) {
- 
-  if (!parent.navigator.mozMobileMessage) {
+
+  if (!opener || !opener.navigator.mozMobileMessage) {
```

Comment:
not really sure we should use opener; sorry if I miss something but where will this script run ?